### ISSUE TITLE
Does the include must not exlude everything else

### ISSUE THREAD

**Issue by Barbapapazes** [harlan-zw/nuxt-simple-sitemap#114]
### Describe the bug

When doing manual chunking, using include does not exclude every things else.

When I add the include rule, I expect that everything else is exclude. Otherwise, I need to exclude each other sitemap which is annoying when having many sitemaps.

### Reproduction

/

### System / Nuxt Info

```Shell
------------------------------
- Operating System: Linux
- Node Version:     v18.16.1
- Nuxt Version:     3.6.5
- Nitro Version:    2.5.2
- Package Manager:  pnpm@8.6.11
- Builder:          vite
- User Config:      app, modules, css, nitro, content, unocss, sitemap, devtools
- Runtime Modules:  @nuxt/content@2.7.2, @vueuse/nuxt@10.3.0, @unocss/nuxt@0.53.6, nuxt-simple-robots@3.0.3, nuxt-simple-sitemap@3.1.3
- Build Modules:    -
------------------------------
```

**Comment by Barbapapazes** [comment]
In reality, I'm not able to understand how the sitemap splitting works since using this config, I've got the exact same sitemap for every chunks.

```ts
  sitemap: {
    sitemaps: {
      learn: {
        include: ['/learn/**'],
        exclude: ['/build/**', '/explore/**', '/blog/**', '/categories/**', '/packages/**'],
      },
      build: {
        include: ['/build/**'],
        exclude: ['/explore/**', '/learn/**', '/blog/**', '/categories/**', '/packages/**'],
      },
      explore: {
        include: ['/explore/**'],
        exclude: ['/build/**', '/learn/**', '/blog/**', '/categories/**', '/packages/**'],
      },
      blog: {
        include: ['/blog/**'],
        exclude: ['/build/**', '/explore/**', '/learn/**', '/categories/**', '/packages/**'],
      },
      categories: {
        include: ['/categories/**'],
        exclude: ['/build/**', '/explore/**', '/learn/**', '/blog/**', '/packages/**'],
      },
      packages: {
        include: ['/packages/**'],
        exclude: ['/build/**', '/explore/**', '/learn/**', '/blog/**', '/categories/**'],
      },
      pages: {
        exclude: ['/build/**', '/explore/**', '/learn/**', '/blog/**', '/categories/**', '/packages/**', '/_**'],
      },
    },
  },
```

**Comment by Barbapapazes** [comment]
Even with regex, it does not work. I'm using `nuxt/content` with document driven mode.

**Comment by harlan-zw** [comment]
Thanks, this looks like a bug. Regex isn't supported as it uses radix3 as the filter 

Will take a look, there's a couple of other tickets for properly documenting the filtering so have that on my radar as well.

**Comment by Barbapapazes** [comment]
Regex is not supported? But types show string or Regex!

**Comment by Barbapapazes** [comment]
It seems to be resolved in 3.1.5.

**Comment by harlan-zw** [comment]
Thanks, sorry looks like I forgot to ping you.
